versioned_docs/version-v2.4.1/key-features/device-sharing.md

```
@@ -7,4 +7,4 @@ title: Device sharing
 - Permits partial device allocation by specifying device core usage.
 - Requires zero changes to existing programs.
 
-![HAMi device sharing example showing multiple tasks sharing a single GPU](/img/docs/common/key-features/example.png)
\ No newline at end of file
+![HAMi device sharing example showing multiple tasks sharing a single GPU](/img/docs/en/key-features/example.svg)
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)